telegram/ext/_applicationbuilder.py

```diff
@@ -617,6 +628,26 @@ def write_timeout(self: BuilderType, write_timeout: Optional[float]) -> BuilderT
         self._write_timeout = write_timeout
         return self
 
+    def media_write_timeout(
+        self: BuilderType, media_write_timeout: Optional[float]
+    ) -> BuilderType:
+        """Sets the media write operation timeout for the
```

Comment:
do we want to document the precedence order of choosing the write timeout? i.e. the code block in https://github.com/python-telegram-bot/python-telegram-bot/issues/3864#issuecomment-1712796820. Specifically, we should document that changing `AppBuilder.write_timeout(x).build()` does NOT have an effect on the write timeout for uploading files, i.e. that stays 20 secs by default.

Reply:
The correct place for elaborating the documentation would be `HTTPXRequest` IMO. All the methods in AppBuilder joust point to the respective docs. I'll add a note to `HTTPXRequest.param.write_timeout` pointing out that it doesn't affect files.